docs/folders.md

```diff
@@ -13,6 +13,7 @@ group, and perform other common folder operations (move, copy, delete, etc.).
 * [Move a Folder](#move-a-folder)
 * [Rename a Folder](#rename-a-folder)
 * [Delete a Folder](#delete-a-folder)
+* [Get trashed items](#get-trashed-items)
```

Comment:
Tiny nitpick, but this should match the title below in capitalization: `Get Trashed Items`